code/modules/chemistry/Reagents-Medical.dm

```diff
@@ -627,6 +630,19 @@ datum
 			on_mob_life(var/mob/M, var/method=INGEST, var/mult = 1)
 				if(!M)
 					M = holder.my_atom
+				if(holder.has_reagent("neurotoxin"))
+					holder.remove_reagent("neurotoxin", 3 * mult)
+				if(holder.has_reagent("capulettium"))
+					holder.remove_reagent("capulettium", 3 * mult)
+				if(holder.has_reagent("sulfonal"))
+					holder.remove_reagent("solfonal", 3 * mult)
```

Comment:
typo?

Reply:
One of these days I'm gonna learn to read.

Reply:
Fixed!